docs/kubernetes/getstarted.md

```diff
@@ -0,0 +1,604 @@
+# Kubernetes cluster provisioning, deploiement and examples on Threefold Grid
+
+In this guide we will walk you through the provisioning of a full-blown kubernetes cluster
+on the TF grid.
+
+We will then see how to connect to it and interact using kubectl on our local machine.
+
+Finally we will go through some examples use cases to grasp the features offered by the cluster.
+
+# TL;DR
+
+If you quickly want to setup a kubernetes cluster and start to play with it, you can launch the startup script.
```

Comment:
which startup script ? maybe make a link to the actual file.

Reply:
I added a comment